docs/reference.cue

```diff
@@ -1 +1,36 @@
 package metadata
+
+#Fields: [Name=string]: #Fields | _
+
+remap: {
+  errors: [Name=string]: {
+    description: string
+    name: Name
+  }
+
+  functions: [Name=string]: {
+    arguments: [
+      ...{
+        required: bool,
+
+        if !required {
+          name: string
+        }
+
+        type: "float" | "int" | "string"
```

Comment:
```suggestion
        type: "float" | "int" | "string" | "bool" | "object" | "array" | "null"
```

_we also have a "timestamp", it's technically either a string _or_ an integer, but it's represented as a separate type in remap function signatures, and argument validation will fail if the value can't be coerced into a timestamp_

Reply:
Additionally, a parameter can accept more than one value type, so this would have to be a list.